src/aspire/utils/misc.py

```diff
@@ -274,26 +274,43 @@ def inverse_r(size, x0=0, y0=0, peak=1, dtype=np.float64):
     return (peak / vals).astype(dtype)
 
 
-def fuzzy_mask(L, r0, risetime, origin=None):
+def fuzzy_mask(L, dtype, r0=None, risetime=None):
     """
-    Create a centered 1D to 3D fuzzy mask of radius r0
+    Create a centered 1D to 3D fuzzy mask of radius r0.
 
     Made with an error function with effective rise time.
 
-    :param L: The sizes of image in tuple structure
-    :param r0: The specified radius
-    :param risetime: The rise time for `erf` function
-    :param origin: The coordinates of origin
+    :param L: The sizes of image in tuple structure.
```

Comment:
Ensure that `L` denotes square/cubic size.

Reply:
Good call. Added checks and corresponding tests. Thanks